DESIGN_AND_ROADMAP.md

```diff
@@ -67,7 +67,8 @@ If you require more accuracy in your editor, consider adopting a type system and
 This applies to multiple language server features such as go to definition, hover, completion and automated refactors.
 Consider the following examples:
 
-> [!NOTE] not all of the examples below are supported at the moment and this is not an exhaustive list. Please check the
+> [!NOTE]
+not all of the examples below are supported at the moment and this is not an exhaustive list. Please check the
```

Comment:
The content needs to be quoted too.

Reply:
It seems to appear correctly in the preview without that. 

(I've got a feeling GH have been changing this feature a bit recently)

Reply:
(But it'll be need if we want multiple paragraphs so I'll update it for consistency)